server/src/main/java/io/crate/execution/ddl/tables/MappingUtil.java

```diff
@@ -49,6 +50,8 @@
 
 public final class MappingUtil {
 
+    public static final String DROPPED_COLUMN_NAME_PREFIX = "........cratedb.dropped.";
```

Comment:
As this is internal, we do not need to follow PG here. How these columns exposed could be defined at the relevant tables in `pg_catalog`  and `information_schema`.
Isn't the change at `ObjectMapper` only needed as dots are used? I suggest to use a different prefix like e.g. `_dropped_`.

Reply:
Thanks, it makes it simpler - reverted `ObjectMapper` change and updated prefix in https://github.com/crate/crate/pull/14803/commits/a4ecf22703d53280cd43978b1bea7e236d2e0661

Reply:
Pushed a fixup https://github.com/crate/crate/pull/14803/commits/0f340dbc6934fa40db83f72966889f4ed1eb564c spot by `pg_attrbute` change (will push a dedicated change soon in another PR)